### PR TITLE
Store Playback Status in new table

### LINKED ISSUE #964: [REQUEST] Play Count
It search and read the manual script and for the life of me can not find what I can only assume exists, the play count of a particular item, can someone point me the way, or if its no implemented, can I put in a request plz.

 Thanks for this awesome software I have been abler to produce exactly (almost) the menu system for all my films with it :)

### LINKED ISSUE #1003: [Feature request]: Store the progress of viewed videos
Hi,

maybe its not the right place for this. Pls, move, or suggest me where to submit.

Anyhow, it would be really cool, if possible, to have this feature, so at continuing the watching it will proceed directly and no need of rewinding.

Thanks

### LINKED ISSUE #1642: mark-played-items extended runtime
Not an issue as such, but is there a scope where you can make it client specific in terms of the ip address of last played the item? Instead for all clients?

### PR DESCRIPTION
**Pull request by KarlStraussberger**
Brings support for
```
upnp:playbackCount
upnp:lastPlaybackTime
upnp:lastPlaybackPosition

```
closes #964 
prepares for #1003 (not yet implemented)
closes #1642

### CODE REVIEW

File: src/common.h [hunk 1]
```diff
@@ -92,6 +92,8 @@ static constexpr auto PROTOCOL = std::string_view("http-get");
 #define DEFAULT_COMMON_SCRIPT "common.js"
 #define DEFAULT_WEB_DIR "web"
 #define DEFAULT_JS_DIR "js"
+#define CLIENT_GROUP_TAG "group"
```
Comment (whyman) [inline src/common.h:95]: constexpr string_view?

Comment (KarlStraussberger) [reply src/common.h:95]: Breaks all the assignments to string.

File: src/cds_objects.h [hunk 2]
```diff
@@ -402,6 +404,8 @@ class CdsItem : public CdsObject {
 
     std::chrono::milliseconds bookMarkPos {};
 
+    std::shared_ptr<ClientStatusDetail> playStatus {};
```
Comment (neheb) [inline src/cds_objects.h:407]: shared_ptr is default initialized to nullptr. No need for {}.

**Comment by whyman** [comment]
Ready to merge?

**Comment by KarlStraussberger** [comment]
yes. Test last night ran well.

**Pull request merged**